runtime/src/iree/modules/check/module.cc

```diff
@@ -231,8 +231,9 @@ TransferBuffersToHost(
 static Status TransferToHost(iree_hal_device_t* device,
                              vm::ref<iree_hal_buffer_view_t>& buffer_view) {
   IREE_TRACE_SCOPE();
+  iree::span<const vm::ref<iree_hal_buffer_view_t>> source_views({buffer_view});
```

Comment:
Oh, ASan reports leaks with this code. Logs: https://github.com/openxla/iree/actions/runs/6804836424/job/18503174528?pr=15489#step:4:8584

(probably something about copying/moving...)